visualizer/flatten.js

```diff
@@ -69,11 +113,16 @@ Flatten.prototype.nodes = function () {
   // Flatten out the nodes, removed children of collapsed nodes and calculate
   // index.
   const nodes = [];
```

Comment:
what does this stem from? I don't exactly understand (maybe I don't need to, lol)